app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -19,11 +19,21 @@ def connect(options = {})
   end
 
   # Same as calling connect.
-  # @param _auth_type [nil] Not used
+  # @param _auth_type [nil]
   # @param options [Hash] Connection options.
-  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc]
+  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc] or [Boolean]
   def verify_credentials(_auth_type = nil, options = {})
```

Comment:
If you're going to use the auth_type passed in we should drop the leading `_` as that is an indication to linters (rubocop) that it is okay that the argument is unused.